### ISSUE TITLE
Meaning of Info during training?

### ISSUE THREAD

**Issue by bexx1963** [experiencor/keras-yolo2#239]
What does the info show me during training?

Loss XY 	[0.0642155707] --> Which loss is this?
Loss WH 	[0.493789375]  --> Which loss is this?
Loss Conf 	[0.0898802057] -->Which loss is this?
Loss Class 	[1.10027122]  -->Is this the loss for the classification prediction?
Total Loss 	[1.74815643] --> Total loss of what? Classification and Detection?
Current Recall 	[0.99999994] --> This stays the whole time at 0.99, why?
Average Recall 	[0.993055463]

**Comment by experiencor** [comment]
@bexx1963 Please the YOLO paper. The name of the variable is self-explanatory.

**Comment by bexx1963** [comment]
Which loss is most important for indicating quality of detection network?

**Comment by experiencor** [comment]
It's the current recall.

**Comment by experiencor** [comment]
I really don't know but it should approach zero initially during warmup. It will then go up when the network starts to be able to pick up the object.

**Comment by bexx1963** [comment]
And when there is no warmup? Maybe i can display another key figure such as validation accuracy, how would i calculate it? Or is the keras metrics option in model.fit sufficient?

**Comment by experiencor** [comment]
I do print recall.
